Misc/NEWS

```diff
@@ -303,6 +303,9 @@ Extension Modules
 Library
 -------
 
+- bpo-29649: struct.pack_into() check boundary error message
```

Comment:
Perhaps something like "Exception message of struct.pack_into() now respects [...]" might be better. It's 4am here so perhaps @vadmium can come up with a better wording :)